### DISCUSSION TITLE
［求助］

### DISCUSSION THREAD

**Discussion by llxcx4** [share121/inter-knot#1760] [其他]
![Screenshot_20240812_203844_mark via](https://github.com/user-attachments/assets/774fbd78-34a7-4859-9a20-1c086ce656a9)
今天更新后就成这样子了┐(´-｀)┌

**Comment by llxcx4** [comment]
今天更新完就成这样子了，希望有大佬帮帮忙

**Comment by MiManchi-netizen** [comment]
开加速器

**Reply by llxcx4** [reply]
我这开了也没用啊

**Comment by fairy-B** [comment]
思题目++开了没用的（悲

**Comment by keerdun** [comment]
把浏览器删了重新下就好了
